src/store/getters/state-getters.ts

```diff
@@ -522,12 +527,20 @@ export const isFilingValid = (state: StateIF): boolean => {
     // detail
     // certify (client error correction only)
     // staff payment
-    return (
-      state.stateModel.peopleAndRoles.valid &&
-      state.stateModel.detail.valid &&
-      state.stateModel.certifyState.valid &&
-      state.stateModel.staffPaymentStep.valid
-    )
+    if (state.stateModel.tombstone.correctionType === CorrectionErrorTypes.CLIENT) {
+      return (
+        state.stateModel.peopleAndRoles.valid &&
+        state.stateModel.detail.valid &&
+        state.stateModel.certifyState.valid &&
+        state.stateModel.staffPaymentStep.valid
+      )
+    } else {
+      return (
+        state.stateModel.peopleAndRoles.valid &&
+        state.stateModel.detail.valid &&
+        state.stateModel.staffPaymentStep.valid
+      )
+    }
```

Comment:
Yes, I knew this would change. Good job.

For future: Completing Party will be implemented/updated in ticket #12577. I'm not sure what that will look like and whether it can be valid or not, but we may need to update this code again.